packages/sdk/src/across/constants.ts

```diff
@@ -34,3 +48,12 @@ export const RATE_MODELS = {
     R2: "2000000000000000000",
   },
 };
+
+export function getRateModel(address: string): RateModelBN | undefined {
```

Comment:
do we have to do this? it feels like this is somewhat unnecessary and adding the extra type is a bit confusing.

Reply:
yea true, its just that the function required big nums and i didnt want the constants to be Bignums. i can think of a better way to do this